crates/pop-drink/src/error.rs

```diff
@@ -0,0 +1,258 @@
+//! A set of errors used for testing smart contracts.
+
+use std::fmt::Debug;
+
+pub use drink::{
+	pallet_assets::Error as AssetsError, pallet_balances::Error as BalancesError,
+	pallet_contracts::Error as ContractsError,
+};
+pub use pop_api::primitives::v0::*;
+use scale::{Decode, Encode};
+
+use crate::utils::decode;
+
+/// Configuration for the runtime error.
+pub trait RuntimeErrorConfig: Debug {
```

Comment:
Do we even need this trait when `RuntimeError` can just be updated to take the types as generic parameters? Would eliminate a decent amount of code.

Reply:
Resolved